src/Interactor/AuthorizeCard.php

```diff
@@ -43,7 +44,7 @@ public function handle(OmniCreditCard $omniCreditCard, $ownerId): ?CreditCard
                 ->setCardReference($this->response->getCardReference())
                 ->setCity($omniCreditCard->getBillingCity())
                 ->setCountry($omniCreditCard->getBillingCountry())
-                ->setExpirationDate(new DateTime($omniCreditCard->getExpiryDate('Y-m-y')))
+                ->setExpirationDate(new DateTime($omniCreditCard->getExpiryDate('Y-m-d')))
```

Comment:
@iampersistent Here is the main fix for this pull request. Rest of the changes can be ignored. I've also added corresponding UnitTest to make sure 03/32 date.

Reply:
Good catch. Thanks